client/components/WorkqueueContainer/index.tsx

```diff
@@ -16,8 +16,9 @@ export const WorkqueueComponent = ({
     unlockAndCloseEditor,
     openEditForm,
     openConfirmationModal,
+    workQueueOpen,
```

Comment:
You're using `workQueueOpen` everywhere, which the name doesn't make sense.
The name should indicate what the value represents, such as `mainMenuOpen`